README.md

```diff
@@ -42,8 +42,58 @@ Currently supported models are listed below.
 
 -   No humidifiers are supported at this time, but possible support in the future is being discussed in [issue #60](https://github.com/JeffSteinbok/hass-dreo/issues/60)
 
+#### Space Heaters
+
+-   DR-HSH004S
+-   DR-HSH009S / WH719S (no schedule support, no oscillation angle support as of 02/08/24)
+
+Heaters are modeled as climate devices, which enables the ability to put a thermostat control for the heater into your HA dashboards. Some examples are shown below.
+
+
+<table>
+    <tr>
+        <td><img src="https://raw.githubusercontent.com/andy-cooper/hass-dreo/d1e67331ba1be79a8635a6c3aaa54ac279e7a243/auto-mode-thermostat.png" width="300" alt="eco/auto mode thermostat"></td>
```

Comment:
Please ensure the URLs are correct after the merge.

Reply:
Yup. I *thought* I had done those as relative URLs. I'll make sure they're correct after merge.

Reply:
Ah - I know why they're not relative - relative links apparently don't work. Once merged, we'll need to change the links.